### ISSUE TITLE
VS Code, Create new Project, C#, title and description can be made clearer 

### ISSUE THREAD

**Issue by softchris** [Azure/azure-functions-templates#966]
Currently, you create a Durable functions project for C# in VS Code by following a wizard and selecting a task called
`DurableFunctionsOrchestration`. What it in fact does it to give you a `Start`, `Orchestration` and first `Activity` function. That's great, but it's a surprise given its name and lack of a description (and given how it works for JavaScript where this is three separate tasks).

My suggestion:

Rename from `DurableFunctionsOrchestration` to `DurableFunctionsStarter` and add description saying something like this:

```
This task will create a Start function, and Orchestration function, and a first activity function. 
```

**Comment by fabiocav** [comment]
@AnatoliB thoughts on this? This seems to make sense.

**Comment by AnatoliB** [comment]
I agree the current name is confusing. I'm not sure about the proposed name though, as "Starter" may be also interpreted as "the starter function only". @anthonychu, what do you think?

**Comment by anthonychu** [comment]
Agree that it could use a better name but also agree that "starter" would be confusing in this context.

**Comment by liliankasem** [comment]
@AnatoliB @anthonychu any thoughts on next steps to address this, we don't think it's part of the host project but should be tracked somewhere?

**Comment by AnatoliB** [comment]
@softchris Please note that the template is called DurableFunctionsOrchestrat**ion**, and not DurableFunctionsOrchestrat**or**. There is a difference between _orchestrator_ and _orchestration,_ even though these terms look similar:
- [Orchestrator](https://docs.microsoft.com/en-us/azure/azure-functions/durable/durable-functions-types-features-overview#orchestrator-functions) is a function type. One of the functions created by the template will be of this type.
- [Orchestration](https://docs.microsoft.com/en-us/azure/azure-functions/durable/durable-functions-orchestrations) is a more complex entity. I couldn't find a strict definition now, but please check out the link. This term is used to encompass all the involved orchestrator and function definitions (both orchestrator and activity functions), together with the execution context, history, etc. It is definitely not limited by the _orchestrator_ function only.

With this idea in mind, the current name (DurableFunctionsOrchestrat**ion**) does not look that bad. However, I do acknowledge the visual similarity of these two words, and I know that users often treat them as interchangeable. I'm still not convinced this deserves a change.

@cgillum What do you think about improving the template name?

**Comment by cgillum** [comment]
I think I mostly agree with what everyone has said so far. However, I do like the suggestion of "DurableFunctionsStarter". I agree it's ambiguous with the "[client](https://docs.microsoft.com/en-us/azure/azure-functions/durable/durable-functions-types-features-overview#client-functions)" function that's often used to "start" durable orchestrations, but I actually feel it's less ambiguous since there is no "starter" concept today (just the "client") and I assume most users won't have a preconceived notion of what a starter/client function is.

**Comment by andystaples** [comment]
We haven't been able to agree on a better name, so we're electing to leave it as-is for now
